0.14.3 install bug w/ init-status

Anyone else getting this?

```sh
❯ npm i bundlesize -D
npm ERR! path /Users/scoope7/ngx-trend/node_modules/bundlesize/src/init-status.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/scoope7/ngx-trend/node_modules/bundlesize/src/init-status.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/scoope7/.npm/_logs/2017-08-31T17_22_26_445Z-debug.log
```
**Please mention other relevant information.**
- node version  
v8.3.0
- npm version  
5.4.0
- Operating system  
osx
- bundlesize version  
0.14.3
- CI you are using  
travis, but this is on my local


already tried verifying npm cache and then i cleared it just because. Using 0.14.2 for now.

--- a/src/init-status.js
+++ b/src/init-status.js
@@ -1,3 +1,4 @@
 #!/usr/bin/env node
 
+/* eslint-disable no-unused-vars */
 const build = require('./build')

--- a/src/init-status.js
+++ b/src/init-status.js
@@ -1,4 +1,3 @@
 #!/usr/bin/env node
 
-/* eslint-disable no-unused-vars */
-const build = require('./build')
+require('./build')

--- a/src/init-status.js
+++ b/src/init-status.js
@@ -1,0 +1,3 @@
+#!/usr/bin/env node
+
+const build = require('./build')

